src/pkg/utils/bytes.go

```diff
@@ -50,3 +55,35 @@ func ByteFormat(inputNum float64, precision int) string {
 
 	return strconv.FormatFloat(returnVal, 'f', precision, 64) + unit
 }
+
+// CheckLocalProgress continuously checks the local directory size and updates the progress bar.
+// NOTE: This function runs infinitely until the completeChan is triggered.
+func CheckLocalProgress(progressBar *message.ProgressBar, expectedTotal int64, filepath string, wg *sync.WaitGroup, completeChan chan int, updateText string) {
+	for {
+		select {
+		case <-completeChan:
+			// Fill out the progress bar so that it's a little less choppy
+			title := fmt.Sprintf("%s (%s of %s)", updateText, ByteFormat(float64(expectedTotal), 2), ByteFormat(float64(expectedTotal), 2))
+			progressBar.Update(int64(expectedTotal), title)
+
+			// Send success message
+			progressBar.Successf("%s (%s)", updateText, ByteFormat(float64(expectedTotal), 2))
+			wg.Done()
```

Comment:
This seems to result in some visual glitches on package create
![image](https://user-images.githubusercontent.com/3977569/223589294-926817c2-017d-4571-b2c8-343048677877.png)